### PR TITLE
chore(console): add e2e tests

### PR DESCRIPTION
**Pull request by skyrpex**
- The console uses https://playwright.dev/ to run e2e tests
  - Uses `data-testid` parameters to locate elements
- Uses the [IBM Plex font](https://github.com/IBM/plex)
- The `wing it` command uses port 1214 by default, if available
- Adds an optional `--wingfile` parameter to the console dev and preview scripts

Resolves [#2929](https://github.com/winglang/wing/issues/2929).

### CODE REVIEW

File: apps/wing-console/console/app/.gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+*.png binary
```
Comment (MarkMcCulloh) [inline apps/wing-console/console/app/.gitattributes:1]: Is this really not the default in git?
If not, might as well put this in the root .gitattributes instead.

File: apps/wing-console/console/app/scripts/dev.mjs [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
 import { createServer as createViteServer } from "vite";
 import { viteConfig } from "./config.mjs";
 
+const options = parseArgs({
+  options: {
+    wingfile: {
+      type: "string",
+    },
+  },
+});
```
Comment (MarkMcCulloh) [inline apps/wing-console/console/app/scripts/dev.mjs:15]: 💯

**Review by MarkMcCulloh — APPROVED** [review]
LGTM, glad it didn't add much time to the CI. 
I'll add do-not-merge in case you'd like the rest of the console team to take a look.

File: apps/wing-console/console/app/scripts/preview.mjs [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
+});
+
 const { port } = await createConsoleApp({
-  wingfile: fileURLToPath(
-    new URL("../../desktop/demo/index.w", import.meta.url),
-  ),
+  wingfile:
+    options.values.wingfile ??
+    fileURLToPath(new URL("../demo/index.w", import.meta.url)),
+  requestedPort: 3000,
```
Comment (staycoolcall911) [inline apps/wing-console/console/app/scripts/preview.mjs:20]: Port 3000 is usually used by docker. Perhaps it's better to use a different number for default

Comment (ainvoner) [reply apps/wing-console/console/app/scripts/preview.mjs]: https://www.gematrix.org/?word=wing+console

Wing Console in [Hebrew Gematria](https://www.gematrix.org/?word=wing+console#Hebrew-results) equals 1214

**Review by ainvoner — APPROVED** [review]

**Pull request merged**